### DISCUSSION TITLE
The discord link is dead

### DISCUSSION THREAD

**Discussion by CLOVIS-AI** [giraud/reasonml-idea-plugin#307] [General]
Just creating this thread to 'un-pollute' the issue tracker, it replaces #306

**Comment by CLOVIS-AI** [comment]
I think this should create a unlimited link: https://support.discord.com/hc/en-us/articles/208866998-Invites-101

**Comment by giraud** [comment]
does that work ? -> https://discord.gg/65fz5jb

**Reply by CLOVIS-AI** [reply]
Yes, it does, thanks :)

**Reply by giraud** [reply]
updated docs
